2023-06-06 23:28:48,814 ERROR: Exception on /reset_password_request [POST] [in /home/anton/code_train/Flask/flask/lib/python3.11/site-packages/flask/app.py:1414]
Traceback (most recent call last):
  File "/home/anton/code_train/Flask/flask/lib/python3.11/site-packages/flask/app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anton/code_train/Flask/flask/lib/python3.11/site-packages/flask/app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anton/code_train/Flask/flask/lib/python3.11/site-packages/flask/app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anton/code_train/Flask/flask/lib/python3.11/site-packages/flask/app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anton/code_train/Flask/blog_1/app/routes.py", line 173, in reset_password_request
    send_password_reset_email(user)
  File "/home/anton/code_train/Flask/blog_1/app/email.py", line 17, in send_password_reset_email
    token = user.get_reset_password_token()
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anton/code_train/Flask/blog_1/app/models.py", line 65, in get_reset_password_token
    app.config['SECRET_KEY'], algorithm='HS256').decode('utf-8')
                                                 ^^^^^^
AttributeError: 'str' object has no attribute 'decode'
2023-06-06 23:33:05,692 INFO: Microblog startup [in /home/anton/code_train/Flask/blog_1/app/__init__.py:58]
2023-06-06 23:33:10,091 ERROR: Exception on /reset_password_request [POST] [in /home/anton/code_train/Flask/flask/lib/python3.11/site-packages/flask/app.py:1414]
Traceback (most recent call last):
  File "/home/anton/code_train/Flask/flask/lib/python3.11/site-packages/flask/app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anton/code_train/Flask/flask/lib/python3.11/site-packages/flask/app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anton/code_train/Flask/flask/lib/python3.11/site-packages/flask/app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anton/code_train/Flask/flask/lib/python3.11/site-packages/flask/app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anton/code_train/Flask/blog_1/app/routes.py", line 173, in reset_password_request
    send_password_reset_email(user)
  File "/home/anton/code_train/Flask/blog_1/app/email.py", line 17, in send_password_reset_email
    token = user.get_reset_password_token()
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anton/code_train/Flask/blog_1/app/models.py", line 65, in get_reset_password_token
    app.config['SECRET_KEY'], algorithm='HS256').decode('utf-8')
                                                     ^^^^^^
AttributeError: 'str' object has no attribute 'decode'
2023-06-06 23:33:24,400 INFO: Microblog startup [in /home/anton/code_train/Flask/blog_1/app/__init__.py:58]
2023-06-06 23:33:27,124 ERROR: Exception on /reset_password_request [POST] [in /home/anton/code_train/Flask/flask/lib/python3.11/site-packages/flask/app.py:1414]
Traceback (most recent call last):
  File "/home/anton/code_train/Flask/flask/lib/python3.11/site-packages/flask/app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anton/code_train/Flask/flask/lib/python3.11/site-packages/flask/app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anton/code_train/Flask/flask/lib/python3.11/site-packages/flask/app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anton/code_train/Flask/flask/lib/python3.11/site-packages/flask/app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anton/code_train/Flask/blog_1/app/routes.py", line 173, in reset_password_request
    send_password_reset_email(user)
  File "/home/anton/code_train/Flask/blog_1/app/email.py", line 17, in send_password_reset_email
    token = user.get_reset_password_token()
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anton/code_train/Flask/blog_1/app/models.py", line 65, in get_reset_password_token
    app.config['SECRET_KEY'], algorithm='HS256')
                                                 
AttributeError: 'str' object has no attribute 'decode'
2023-06-06 23:33:37,172 INFO: Microblog startup [in /home/anton/code_train/Flask/blog_1/app/__init__.py:58]
2023-06-06 23:33:43,425 INFO: Microblog startup [in /home/anton/code_train/Flask/blog_1/app/__init__.py:58]
2023-06-06 23:33:43,793 INFO: Microblog startup [in /home/anton/code_train/Flask/blog_1/app/__init__.py:58]
2023-06-06 23:34:14,916 ERROR: Exception on /reset_password_request [POST] [in /home/anton/code_train/Flask/flask/lib/python3.11/site-packages/flask/app.py:1414]
Traceback (most recent call last):
  File "/home/anton/code_train/Flask/flask/lib/python3.11/site-packages/flask/app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anton/code_train/Flask/flask/lib/python3.11/site-packages/flask/app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anton/code_train/Flask/flask/lib/python3.11/site-packages/flask/app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anton/code_train/Flask/flask/lib/python3.11/site-packages/flask/app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anton/code_train/Flask/blog_1/app/routes.py", line 173, in reset_password_request
    send_password_reset_email(user)
  File "/home/anton/code_train/Flask/blog_1/app/email.py", line 17, in send_password_reset_email
    token = user.get_reset_password_token()
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anton/code_train/Flask/blog_1/app/models.py", line 65, in get_reset_password_token
    app.config['SECRET_KEY'], algorithm='HS256').decode('utf-8')
                                                 ^^^^^^
AttributeError: 'str' object has no attribute 'decode'
2023-06-06 23:34:38,288 INFO: Microblog startup [in /home/anton/code_train/Flask/blog_1/app/__init__.py:58]
2023-06-06 23:35:02,274 INFO: Microblog startup [in /home/anton/code_train/Flask/blog_1/app/__init__.py:58]
2023-06-06 23:35:11,066 ERROR: Exception on /reset_password_request [POST] [in /home/anton/code_train/Flask/flask/lib/python3.11/site-packages/flask/app.py:1414]
Traceback (most recent call last):
  File "/home/anton/code_train/Flask/flask/lib/python3.11/site-packages/flask/app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anton/code_train/Flask/flask/lib/python3.11/site-packages/flask/app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anton/code_train/Flask/flask/lib/python3.11/site-packages/flask/app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anton/code_train/Flask/flask/lib/python3.11/site-packages/flask/app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anton/code_train/Flask/blog_1/app/routes.py", line 173, in reset_password_request
    send_password_reset_email(user)
  File "/home/anton/code_train/Flask/blog_1/app/email.py", line 17, in send_password_reset_email
    token = user.get_reset_password_token()
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anton/code_train/Flask/blog_1/app/models.py", line 65, in get_reset_password_token
    app.config['SECRET_KEY'], algorithm='HS256')
                                             ^^^^
AttributeError: 'str' object has no attribute 'decode'
2023-06-06 23:35:55,041 INFO: Microblog startup [in /home/anton/code_train/Flask/blog_1/app/__init__.py:58]
2023-06-06 23:36:21,124 INFO: Microblog startup [in /home/anton/code_train/Flask/blog_1/app/__init__.py:58]
2023-06-06 23:36:25,469 INFO: Microblog startup [in /home/anton/code_train/Flask/blog_1/app/__init__.py:58]
2023-06-06 23:36:27,665 INFO: Microblog startup [in /home/anton/code_train/Flask/blog_1/app/__init__.py:58]
